src/AutoPilotPlugins/PX4/CameraComponentSummary.qml

```diff
@@ -14,7 +14,7 @@ FactPanel {
     QGCPalette { id: qgcPal; colorGroupEnabled: enabled }
     FactPanelController { id: controller; factPanel: panel }
 
-    property Fact _camTriggerMode:  controller.getParameterFact(-1, "TRIG_MODE")
+    property Fact _camTriggerMode:  controller.getParameterFact(-1, "TRIG_MODE", false)
```

Comment:
Yes, that would indeed be the better change. If the param is not present no trigger controls should be in place.